microsoft-edge/origin-trials.md

```diff
@@ -0,0 +1,106 @@
+---
+ms.assetid: 
+description: Safely experiment for a fixed period of time and provide feedback on new platform features.
+title: Origin Trials
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 06/29/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: edge, web development, html, css, origin trials, developer
+---
+
+# Use Origin Trials in Microsoft Edge  
+
+Developers may use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you do not need to go to `edge://flags`and turn on feature flags.  For more information, see [experimental APIs][DeveloperMicrsoftEdgeOriginTrials].  Additionally, you may provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.  
+
+## Get started using Origin Trials  
+
+For more information about the experimental APIs available in Microsoft Edge, see [Microsoft Edge Origin Trials Developer Console][DeveloperMicrsoftEdgeOriginTrials].  Ensure that you review the minimum version requirements for Microsoft Edge, and the trial end date to assess suitability of using the experimental APIs on your website.  
+
+> [!NOTE]
+> An experiment may end earlier than planned if any of the following situations occur.  
+> *   A major security incident.  
+> *   If sufficient feedback is collected that indicates a major redesign is needed to meet the needs of web developers.  
+> In either case, a notification email is sent to all developers currently enrolled in the experiment.  
+
+### Register for a trial of an experimental API  
+
+Use the following steps to register for a trial of an experimental API.  
+
+1.  Visit the [Microsoft Edge Origin Trials Developer Console][DeveloperMicrsoftEdgeOriginTrials] page.  
+1.  Choose the Register button on any of the available experiments.  
+1.  Sign into the Developer Console using your GitHub username and password.  
+1.  Choose **Authorize MicrosoftEdge**.  
+1.  Complete the form.  
+    
+    > [!NOTE]
+    > To enroll a single or all subdomains, choose set the `Do you need to match all subdomains for the provided origin?` setting to `Yes`.  For example, `https://dev.contoso.com` is a single domain, and `https://*.contoso.com` uses a wildcard to represent all subdomains.  
+    
+    > [!IMPORTANT]
+    > The following origin formats are not allowed.  
+    > *   Specifying a subfolder on the origin.  
+    >     
+    >     ```https
+    >     https://contoso.com/path/subfolder
```

Comment:
Should we remove this code block and replace with "...on the origin. For example..."?

Reply:
Updated.